.github/workflows/ci.yml

```diff
@@ -26,8 +26,6 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          submodules: true
-          token: ${{ secrets.KROMA_GITHUB_TOKEN }}
```

Comment:
In this `ci.yml` file, `submodules: true` part still exists in `kroma-node`, `kroma-validator`, `kroma-batcher`. Don't we need to remove them?